checks/evaluation/contributors.go

```diff
@@ -16,60 +16,69 @@ package evaluation
 
 import (
 	"fmt"
-	"sort"
 	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/contributorsFromOrgOrCompany"
 )
 
 const (
-	minContributionsPerUser    = 5
 	numberCompaniesForTopScore = 3
 )
 
 // Contributors applies the score policy for the Contributors check.
-func Contributors(name string, dl checker.DetailLogger,
-	r *checker.ContributorsData,
+func Contributors(name string,
+	findings []finding.Finding,
+	dl checker.DetailLogger,
 ) checker.CheckResult {
-	if r == nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, "empty raw data")
+	expectedProbes := []string{
+		contributorsFromOrgOrCompany.Probe,
+	}
+
+	if !finding.UniqueProbesEqual(findings, expectedProbes) {
+		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	entities := make(map[string]bool)
+	numberOfPositives := getNumberOfPositives(findings)
+	reason := fmt.Sprintf("project has %d contributing companies or organizations", numberOfPositives)
 
-	for _, user := range r.Users {
-		if user.NumContributions < minContributionsPerUser {
-			continue
-		}
+	if numberOfPositives >= numberCompaniesForTopScore {
+		// Return max score. This may need changing if other probes
+		// are added for other contributors metrics. Right now, the
+		// scoring is designed for a single probe that returns true
+		// or false.
+		logFindings(findings, dl)
```

Comment:
bit of a nit, but previously we logged as long as number of companies was greater than 0. so this should probably be its own statement above:

```
if numberOfPositives > 0 {
    logFindings(findings, dl)
}
if numberOfPositives > numberCompaniesForTopScore {
    return checker.CreateMaxScoreResult(name, reason)
}
```

Reply:
Done in https://github.com/ossf/scorecard/pull/3460/commits/02d56d541c4ab225fb4541fba32cade1ad0bfcfb